_inc/client/security/jetpack-backup.jsx

```diff
@@ -0,0 +1,132 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { FEATURE_SITE_BACKUPS_JETPACK } from 'lib/plans/constants';
+
+export class JetpackBackup extends Component {
+	static propTypes = {
+		siteRawUrl: PropTypes.string,
+		rewindState: PropTypes.string,
+	};
+
+	static defaultProps = {
+		siteRawUrl: '',
+		rewindState: '',
+	};
+
+	getVaultPressContent = () => {
+		return (
+			<SettingsGroup
+				module={ { module: 'backups' } }
+				support={ {
+					text: __(
+						'Backs up your site to the global WordPress.com servers, ' +
+							'allowing you to restore your content in the event of an emergency or error.'
+					),
+					link: 'https://help.vaultpress.com/get-to-know/',
```

Comment:
Do we still want to send the user to vaultpress.com?

Reply:
Good call!